### PR TITLE
Release Finschia v2.0.2

### PR DESCRIPTION
**Pull request by tkxkd0159**
## Description

closes: #XXXX

## Checklist:

- [x] I followed the [contributing guidelines](https://github.com/Finschia/finschia/blob/main/CONTRIBUTING.md) and [code of conduct](https://github.com/Finschia/finschia/blob/main/CODE_OF_CONDUCT.md).
- [x] I have added a relevant changelog to `CHANGELOG.md`
- [ ] I have added tests to cover my changes.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Review by zemyblue — COMMENTED** [review]
Please check swagger api version. (./client/docs/config.json)
and update RELEASE_NOTE.md

File: RELEASE_NOTE.md [hunk 1]
```diff
@@ -1,32 +1,32 @@
 # Finschia v2.0.1 Release Note
```
Comment (zemyblue) [inline RELEASE_NOTE.md:1]: 2.0.2

**Review by zemyblue — COMMENTED** [review]
Please run `make proto-swagger-gen` and then `make update-swagger-docs`.

**Comment by tkxkd0159** [comment]
This PR will be merged after #336

File: RELEASE_CHANGELOG.md [hunk 2]
```diff
@@ -34,6 +34,13 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 # Changelog
 
+## [v2.0.2] - 2024-03-05
+
+This version base on [finschia-sdk v0.48.1](https://github.com/Finschia/finschia-sdk/releases/tag/v0.48.1), [Ostracon v1.1.3](https://github.com/Finschia/ostracon/tree/v1.1.3), [finschia/wasmd v0.3.0](https://github.com/Finschia/wasmd/releases/tag/v0.3.0) and [finschia/ibc-go v4.3.1](https://github.com/Finschia/ibc-go/releases/tag/v4.3.1).
```
Comment (zemyblue) [inline RELEASE_CHANGELOG.md:39]: I think the used wasmd version is v0.2.0 in finschia v2.0.2.  Please check it.

**Review by ulbqb — APPROVED** [review]

**Pull request merged**